core/activity/src/http_proxy.rs

```diff
@@ -74,8 +80,32 @@ async fn proxy_http_request(
             .call_streaming(msg)
     });
 
+    if let Some(value) = request.headers().get(header::ACCEPT) {
+        if value.eq(mime::TEXT_EVENT_STREAM.essence_str()) {
```

Comment:
Could it be an `APPLICATION_OCTET_STREAM`?

Reply:
added